MvvmCross/Droid/Droid/Platform/MvxAndroidSetup.cs

```diff
@@ -158,7 +159,7 @@ protected virtual void RegisterBindingBuilderCallbacks()
             Mvx.CallbackWhenRegistered<IMvxNamespaceListViewTypeResolver>(FillNamespaceListViewTypeResolver);
         }
 
-        protected virtual MvxAndroidBindingBuilder CreateBindingBuilder()
+        protected virtual MvxBindingBuilder CreateBindingBuilder()
```

Comment:
This leads me to a question...  Should we have a MvxFormsSetup base class?  Not sure if it's possible without reqriting too much code...